### ISSUE TITLE
Issue with "Using the alert role": Missing compatibility data

### ISSUE THREAD

**Issue by Ryuno-Ki** [mdn/content#4321]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role

#### What information was incorrect, unhelpful, or incomplete?

> TBD: Add support information for common UA and AT product combinations

#### Specific section or headline?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role#compatibility

#### What did you expect to see?

Either the section removed or some data (even if incomplete) with OS / Browser / AT versions.

#### Did you test this? If so, how?

I opened the link in Firefox for Sabayon Linux.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/aria/aria_techniques/using_the_alert_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/aria_techniques/using_the_alert_role/index.html
* Last commit: https://github.com/mdn/content/commit/8336f6abbfba98554001d9b9f3a071a14a0e2b1a
* Document last modified: 2021-02-24T07:39:57.000Z

</details>

**Comment by estelle** [comment]
We've updated the alert role at https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role, but removed the compact data and we don't have compact data for assistive technologies.
